tests/integ/test_stored_procedure.py

```diff
@@ -1397,3 +1397,20 @@ def return_success(session_):
             )
             return
         raise
+
+
+def test_force_inline_code(session):
+    large_str = "snow" * 10000
+
+    def f(session: Session) -> int:
+        return len(large_str)
+
+    with session.query_history() as query_history:
+        _ = session.sproc.register(f, packages=["snowflake-snowpark-python"])
+    assert all("AS $$" not in query.sql_text for query in query_history.queries)
+
+    with session.query_history() as query_history:
+        _ = session.sproc.register(
+            f, packages=["snowflake-snowpark-python"], force_inline_code=True
```

Comment:
I'm wondering if the way to set this param through public api should be with an internal parameter name like `session.sproc.register(..., _force_inline_code=True)`.

Reply:
This parameter is added to kwargs, so it won't show up in documents and isn't exposed to users. I follow the same approach as we did for `anonymous` parameter.

Reply:
sounds good. 👍